tests/node-tests/basic-node-tests.js

```diff
@@ -39,8 +39,23 @@ fluid.require("test-module", require, "test-module");
 
 fluid.setLogging(true);
 
+// Set boolean for the  presence of the --tap command line option to output a
+// TAP report for consumption by testem when used as a custom launcher
+var shouldOutputTAP = process.argv.findIndex(function (argument) {
+    return argument.indexOf("--tap") > -1;
+}) > -1;
+
+// For accumulating TAP messages for parsing by testem custom launcher
+var tapOutput = "";
+
 QUnit.testDone(function (data) {
-    fluid.log("Test concluded - " + data.name + ": " + data.passed + " passed, " + data.failed + " failed");
+    var testMessage = "Test concluded - " + data.name + ": " + data.passed + " passed, " + data.failed + " failed";
+    fluid.log(testMessage);
+    if (shouldOutputTAP) {
```

Comment:
Yes, please.  I tried briefly integrating with a third-party plugin but was stymied by our older forked version of QUnit.  I have an approach to get TAP output from QUnit browser tests, I would love to see TAP support available for node jqUnit tests as well.